sc/aml-markers.tab.info

```diff
@@ -0,0 +1,24 @@
+{
+    "collection": "10x Genomics",
+    "description": "Cell type markers for bone marrow cells. Markers were manually extracted from the <a href='https://www.nature.com/articles/ncomms14049'>publication on bone marrow mononuclear cells with AML</a>.",
+    "instances": 8,
+    "name": "aml-markers",
+    "references": [
+        "Zheng GX, Terry JM, Belgrader P et al. (2017) <a href='https://www.nature.com/articles/ncomms14049'> Massively parallel digital transcriptional profiling of single cells, Nat Commun 8:14049.</a>"
+    ],
+    "seealso": [],
+    "size": 308,
+    "source": "<a href='https://www.nature.com/articles/ncomms14049'>10x Genomics</a>",
+    "tags": [
+        "aml",
+        "expression",
+        "markers",
+        "human"
+    ],
+    "target": "none",
+    "title": "Bone marrow mononuclear cells with AML (markers)",
+    "url": "http://bubble.fri.uni-lj.si/zp/singlecell/datasets/aml-markers.tab",
+    "variables": 1,
```

Comment:
This file has 3 columns, so I think this field should be set to 3.

Although the current README says independent variables, I think all variables should be reported, no matter what role they have in Orange domains. Currenlty class variable is included in the count (see e.g. iris, titanic, ...) which is also in conflict with the readme (which needs to be updated IMO).

Reply:
Proposed README change in #37